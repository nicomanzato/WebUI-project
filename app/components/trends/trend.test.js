import React from 'react';
import Trend from './trend'
import TrendMock from './../../store/trend/mock/trendMock'
import { shallow, mount, ReactWrapper } from 'enzyme'

describe('trend', () => {

  it('should render correctly', () => {
    const wrapper = shallow(
      <Trend
        onPress={() => {}}
        trend={TrendMock}
        key={4}
        index={4}
      />
    );
    
    expect(wrapper).toMatchSnapshot();
  });

});
