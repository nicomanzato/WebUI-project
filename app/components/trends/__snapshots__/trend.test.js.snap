// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trend should render correctly 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  delayPressOut={100}
  key="t4"
  onPress={[Function]}
  underlayColor="#dddddd"
>
  <Component
    key="4"
    style={
      Object {
        "borderBottomWidth": 1,
        "borderColor": "#ccc",
        "flexDirection": "row",
        "padding": 10,
      }
    }
  >
    <Component
      style={
        Object {
          "flex": 0.1,
          "paddingLeft": 7,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        key="#4"
        style={
          Object {
            "color": "grey",
            "fontSize": 18,
          }
        }
      >
        5
      </Text>
    </Component>
    <Component
      style={
        Object {
          "flex": 0.9,
          "paddingLeft": 10,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        key="4Name"
        style={
          Object {
            "fontSize": 17,
            "marginBottom": 5,
          }
        }
      >
        Sof√≠a Herrera
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "grey",
            "fontSize": 15,
          }
        }
      >
        11111
         Tweets
      </Text>
    </Component>
  </Component>
</TouchableHighlight>
`;
